src/demo-app/datepicker/datepicker-demo.html

```diff
@@ -7,27 +7,35 @@
 <p>
   <md-input-container>
     <input mdInput [mdDatepicker]="minDatePicker" [(ngModel)]="minDate" placeholder="Min date">
-    <button mdSuffix [mdDatepickerToggle]="minDatePicker"></button>
+    <button md-icon-button mdSuffix [mdDatepickerToggle]="minDatePicker">
+      <md-datepicker-toggle-icon></md-datepicker-toggle-icon>
+    </button>
```

Comment:
What do you think of just having 
```html
<md-datepicker-toggle for="minDatePicker"></md-datepicker-toggle>
```

If someone wants to have a custom icon, they just do a custom `<button>` with the close behavior. `[mdDatepickerToggle]="minDatePicker"` is _longer_ than `(click)="minDatePicker.toggle()"`, so it seems like there's not much sense in breaking up the toggle behavior and the baked-in icon. 

For the baked-in version, all the things someone would typically want to set on the native `<button>` are taken care of by the component (content, aria-label, and type).